### DISCUSSION TITLE
Keyball39's Trackall stopped working

### DISCUSSION THREAD

**Discussion by 619frank** [Yowkees/keyball#721] [General]
After updating the firmware on my Keyball39, the trackball is no longer working.

It functioned correctly for a few days before failing during a later firmware update.

I've tried reinstalling several firmware versions, including the default keymap and one from the Remap website, but the problem persists.

All other keys work fine.

When I connect it via USB, a faint red light appears for about five seconds, then goes out.  A blinking light, too dim to see without a phone camera, is then visible.

Is this normal behavior?

Could you help me determine if the problem is with the sensor or its circuit board?

 Please suggest some troubleshooting and let me know how to proceed further 🙏

**Comment by 619frank** [comment]
https://github.com/user-attachments/assets/cae3d4dc-42a6-4861-a518-0466976cf5d1

**Comment by 619frank** [comment]
https://github.com/user-attachments/assets/18652580-d16d-49ee-883f-e02343287b58

**Comment by 619frank** [comment]
![20241227_163229~3.jpg](https://github.com/user-attachments/assets/0cf1efa2-1020-462b-8777-153b83ce9dd3)

![20241227_163250~2.jpg](https://github.com/user-attachments/assets/e4fd3b87-f4dc-41cd-b2a4-c85ef5ba1872)

![20241227_162959~2.jpg](https://github.com/user-attachments/assets/e4b87dce-93f9-4cd6-8257-d325bdf6865c)

![20241227_163229~2.jpg](https://github.com/user-attachments/assets/c43eb292-63ec-446a-a0c0-85e3828c16a4)

**Comment by jtuyen** [comment]
What does the trackball coordinates values show on the OLED when you move the trackball?

If you aren't getting coordinates at all and you haven't changed anything hardware wise, it's likely the sensor I'm guessing. I had a similar issue when I had a problem between my TRRS connection.

**Reply by 619frank** [reply]
I am not using the OLED screen.

I tried debugging using the QMK console logs. No logs were printed when I moved the trackball.

Should I try replacing the sensor?

**Reply by jtuyen** [reply]
I would think It's worth a shot replacing it. Replacement sensors are relatively cheap to find.

**Reply by 619frank** [reply]
In india, it is hard to source any electronic part😔😞😣

**Comment by Yowkees** [comment]
Hi. Thank you for choosing Keyball39.

I want to discuss how to sell new parts, etc.
Please give me a message from below link.
https://shirogane-lab.net/inquiry/

Thank you very much.
